pypasta/LinuxMailCharacteristics.py

```diff
@@ -152,12 +151,12 @@ def __init__(self, c):
         # Metric: All lists + at least one maintainer per section
         # needs to be addressed correctly
         if (not c.mtrs_has_lists or c.mtrs_has_list_per_section) and \
-           (not c.mtrs_has_maintainers or c.mtrs_has_maintainer_per_section):
+                (not c.mtrs_has_maintainers or c.mtrs_has_maintainer_per_section):
             self.all_lists_one_mtr_per_sub = True
 
         # Metric: At least one correct list + at least one correct maintainer
         if (not c.mtrs_has_lists or c.mtrs_has_one_correct_list) and \
-           (not c.mtrs_has_maintainers or c.mtrs_has_one_correct_maintainer):
+                (not c.mtrs_has_maintainers or c.mtrs_has_one_correct_maintainer):
```

Comment:
it this the "correct" indentation?

Reply:
I did reformat code in pycharm, which has default settings for python.